end-to-end-test/remote/specs/core/screenshot.spec.js

```diff
@@ -165,8 +165,7 @@ function runResultsTestSuite(prefix, options = {}) {
     it(`${prefix} pathwaymapper tab`, function() {
         browser.click('a.tabAnchor_pathways');
         browser.waitForVisible('#cy', 10000);
-        browser.waitForExist('.Toastify__toast', 4000);
-        browser.waitUntil(() => !$('.Toastify__toast').isExisting());
+        browser.waitForExist('#pathway-mapper-message-box', 4000);
```

Comment:
We should use `data-test` prop here:

```suggestion
        browser.waitForExist('div[data-test="pathwayMapperMessageBox"]', 4000);
```